efi/preinstall/errors.go

```diff
@@ -75,3 +81,50 @@ var (
 	// flag is supplied to RunChecks and the current environment is a virtual machine.
 	ErrUEFIDebuggingEnabled = errors.New("the platform firmware contains a debugging endpoint enabled")
 )
+
+// Errors related to checking the TPM device.
```

Comment:
I didn't do this because the top-level `RunChecks` API adds error types that wrap the errors returned from each of these functions, so errors returned from this function get wrapped in an outer `TPM2Error` type, which I was going to group with the errors defined by this var stanza.